src/applications/mhv/secure-messaging/tests/e2e/sm_site/SecureMessagingSite.js

```diff
@@ -67,10 +67,17 @@ class SecureMessagingSite {
       `/my_health/v1/messaging/folders/0/threads?pageSize=10&pageNumber=${interceptedPage}&sortField=SENT_DATE&sortOrder=DESC`,
       mockMessages,
     ).as(`inboxMessagesessages${interceptedPage}`);
-    cy.get('[aria-label="Pagination"]')
-      .shadow()
-      .find(`[aria-label="Page ${interceptedPage}"]`)
-      .click();
+    if (interceptedPage === 1) {
+      cy.get('[aria-label="Pagination"]')
+        .shadow()
+        .find('[aria-label="Page 1, first page"')
+        .click();
+    } else {
```

Comment:
This test seemed to be failing because the text match for the first page of the pagination component also includes "first page" so I added a condition here to check for that:

![Screenshot 2023-10-18 at 1 16 29 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/872479/c74be3e6-e83e-4d7d-9b8d-f92bae4e2cd5)